template/package.json

```diff
@@ -48,9 +48,7 @@
       "output": "build"
     },
     "files": [
-      "dist/electron",
-      "node_modules/",
-      "package.json"
+      "dist/electron/**/*"
```

Comment:
Yes, electron-builder will normalise `dir` to `dir/**/*` automatically, but in the boilerplate it is better to use standard form.